src/foam/support/view/TicketSummaryView.js

```diff
@@ -0,0 +1,157 @@
+    width: 100px;
+    height: 100px;
+    border-radius: 2px;
+    background-color: #59a5d5;
+    display: inline-block;
+  }
+  ^ .Mentions { 
+    font-family: Roboto;
+    font-size: 16px;
+    font-weight: bold;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: 1.25;
+    letter-spacing: 0.3px;
+    text-align: center;
+    color: #ffffff;
+    margin-top: 30px;
+    margin-bottom: 5px;
+  }
+  ^ .M {
+    font-family: Roboto;
+    font-size: 14px;
+    font-weight: normal;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: 0.86;
+    letter-spacing: 0.2px;
+    text-align: center;
+    color: #ffffff;
+  }
+  ^ .ticketdiv{
+    margin: 30px 0px;
+  }
+  ^ .foam-support-view-SummaryCard {
+    width: 170px;
+  }
+  `,
+  messages: [
+    { name: 'title',          message: 'Tickets' },
+    { name: 'newLabel',      message: 'New' },
+    { name: 'updatedLabel',       message: 'Updated' },
+    { name: 'openLabel',       message: 'Open' },
+    { name: 'pendingLabel', message: 'Pending' },
+    { name: 'solvedLabel',      message: 'Solved' }
+  ],
+  properties: [
+    {
+      name: 'dao',
+      factory: function() { return this.user.tickets; }
+    },
+    {
+      class: 'Int',
+      name: 'newCount',
+      value: "...",
+   
+    },
+    {
+      class: 'Int',
+      name: 'updatedCount',
+      value: '...'
+    },
+    {
+      class: 'Int',
+      name: 'openCount',
+      value: '...'
+    },
+    {
+      class: 'Int',
+      name: 'pendingCount',
+      value: '...'
+    },
+    {
+      class: 'Int',
+      name: 'solvedCount',
+      value: '...'
+    },
+    {
+      class: 'Int',
+      name: 'ticketCount',
+      value: '...'
+    },
+    
+  ],
+  methods: [
+    function initE() {
+    this.dao.on.sub(this.onDAOUpdate);    
+    this.onDAOUpdate();
+      this
+        .addClass(this.myClass())
+        .start('div')
+          .start().addClass('ticketdiv')
+            .start().addClass('blue-card-title')
+              .start().add(this.title).addClass('Mentions').end()
+                .start().add(this.ticketCount$).addClass('M').end()           
+            .end()
+            .tag({ class: 'foam.support.view.SummaryCard', count$: this.newCount$, status: this.newLabel })
+            .tag({ class: 'foam.support.view.SummaryCard', count$: this.updatedCount$, status: this.updatedLabel })
+            .tag({ class: 'foam.support.view.SummaryCard', count$: this.openCount$, status: this.openLabel })
+            .tag({ class: 'foam.support.view.SummaryCard', count$: this.pendingCount$, status: this.pendingLabel })
+            .tag({ class: 'foam.support.view.SummaryCard', count$: this.solvedCount$, status: this.solvedLabel })
+          .end()
+        .end()
+    },
+  ],
+  listeners: [
+    {
+      name: 'onDAOUpdate',
+      isFramed: true,
+      code: function() {
+        var self = this;
+        var newDAO = this.dao.where(this.EQ(this.Ticket.STATUS, "New"));
```

Comment:
This could be done more efficiently with a GROUP_BY. But you don't need to do it in this PR. Ask Waseem how.